_includes/footer.html

```diff
@@ -1,7 +1,11 @@
-
-<footer class="footer py-4 js-page-footer">
+<hr>
+<footer class="footer py-4 js-page-footer" >
   <div class="main">
     <center><a href = "../contribute">Click here</a> to contribute an article to learn-pip-trends.com</center>
+    <center>
```

Comment:
Only this part needs to be white. Rest keep as it is because this section and other sections seem like merged.